fiboa_cli/datasets/template.py

```diff
@@ -4,170 +4,143 @@
 # The name of the file will be the name of the converter in the cli.
 # If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
 
-from ..convert_utils import convert as convert_
+from ..convert_utils import BaseConverter
+
+
+class TemplateConverter(BaseConverter):
```

Comment:
I think we should add a comment that explains that the class name also needs to be changed. Do we set any rules for this? e.g. CamelCase of the file name + "Converter"?

Reply:
It's not required but it makes it nicer. Will check in one of the tests... https://github.com/fiboa/cli/pull/138/commits/52fd8bce2432066d009e7520e4793d376d22cf1f